Source/CheckoutTheme.swift

```diff
@@ -1,5 +1,6 @@
 import UIKit
 
+// TODO: This file should have been removed
```

Comment:
Is there something in backlog to remove this ? 

Was this not part of deleting old code. @ehab-al-cko